nbs/cli.ipynb

```diff
@@ -60,6 +60,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        tag = GhApi(gh_host='https://api.github.com').repos.get_latest_release('fastai', 'nbdev-template').tag_name
</pre><p>I am not confident about this change - my main question is are we sure that all other endpoints other than <code>https://api.github.com</code> do not work?  For example, GitHub Enterprise?  </p><p> </p><p>It could be that the OpenAPI spec that ghapi is built on only works with the public GitHub API or something like that, but just curious if you happen to know any more details or have some thoughts around this </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1009/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>